afval.map

```diff
@@ -3108,4 +3108,845 @@ MAP
     END
   END
 
+#-----------------------------------------------------------------------------
+
+  LAYER
+    NAME                    "pand_loopafstand"
+    GROUP                   "afvalcontainers"
+    INCLUDE                 "connection_dataservices.inc"
+    DATA                    "geometrie FROM ( 
+                            SELECT bol.id
+                            , bol.geometrie
+                            , lac.loopafstand_categorie_omschrijving
+                            FROM public.huishoudelijkafval_bag_object_loopafstand bol
+                            INNER JOIN public.huishoudelijkafval_loopafstand_categorie lac
+                            ON bol.loopafstand_categorie_id = lac.id
+                            WHERE 1=1 
```

Comment:
Ik vraag mij af waarom jij WHERE 1=1 toevoegt? Het kan geen kwaad.

Reply:
Oeps, de where 1=1 is gewoonte geworden, om eenvoudiger queries aan te passen om bijvoorbeeld een aantal and's uit te schakelen en vind het overzichtelijker om alleen de and's te voorzien van echter filters